### PR TITLE
Breakdown of Vanilla gui to help with Customization

### PR DESCRIPTION
**Pull request by tenowg**
Fixed all formatting, number of commits, and added come comments.

This can be used by setting the default hud components before vanilla sets them on player login, presumably by calling playerjoinevent event earlier.

Tested locally and all GUI items are present

### CODE REVIEW

**Comment by greatman** [comment]
you need to rebase your commit on top of last edits. It can't be merged.

**Comment by tenowg** [comment]
I believe this did it... git is finicky for me

**Comment by greatman** [comment]
only need to squash your commits so we can import it due to our PR rules. (git rebase -i HEAD~2 in a command line does it)

**Comment by greatman** [comment]
Just tested the commit and it seems working.

**Comment by ghost** [comment]
Your commit doesn't link to your github account.

**Comment by timmhirsens** [comment]
(you can fix that by adding the email adress you used to commit to your github account)

**Comment by tenowg** [comment]
Wish I could say it was as easy as adding the old email address that didn't work anymore... not really sure how that happened... reset everything commited and squash fixed it.

File: src/main/java/org/spout/vanilla/plugin/component/misc/DrowningComponent.java [hunk 1]
```diff
@@ -119,44 +96,25 @@ public void onTick(float dt) {
 					return;
 				}
 				// Animate air meter
+				setAir(5);
```
Comment (Afforess) [inline src/main/java/org/spout/vanilla/plugin/component/misc/DrowningComponent.java:99]: Is this a debug line you forgot to remove?

Comment (tenowg) [inline src/main/java/org/spout/vanilla/plugin/component/misc/DrowningComponent.java:99]: Yes, that should have been removed, maybe a commit got missed while I was fixing everything.

Comment (Afforess) [inline src/main/java/org/spout/vanilla/plugin/component/misc/DrowningComponent.java:99]: Ok. I did some minor cleanup and removed it: https://github.com/VanillaDev/Vanilla/commit/f9f3f10e8888e0cd92276c108094f4c783ae3840#L0L99

**Pull request merged**